apps/webClient/src/views/Home/index.tsx

```diff
@@ -1,62 +1,96 @@
 import React from "react";
 import { makeStyles } from "@material-ui/styles";
-import Box from "@material-ui/core/Box";
 import Grid from "@material-ui/core/Grid";
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
+import Container from "@material-ui/core/Container";
 import Typography from "@material-ui/core/Typography";
 import { Theme } from "@material-ui/core/styles/createMuiTheme";
-import Logo from "../../components/Logo";
+import Logo from "../../components/Logo/";
+import NavButton from "../../components/NavButton";
 
 interface Props {}
 
-const backgroundImage = `${process.env.PUBLIC_BUCKET}/fd-home.jpg`;
-
 const useStyles = makeStyles((theme: Theme) => ({
-  contentWrapper: {
-    paddingBottom: 0,
-    width: 176
-  },
-  top: {
-    backgroundImage: `url("${backgroundImage}")`,
-    height: "100%",
-    backgroundRepeat: "no-repeat",
-    backgroundPosition: "center",
-    backgroundColor: "black"
+  partnerCard: {
+    maxWidth: 300,
+    textAlign: "center",
   },
-  futuredays: {
+  white: {
     color: "white",
-    lineHeight: 1
   },
-  titleContent: {
-    alignSelf: "flex-end",
-    marginBottom: theme.spacing(2),
-    marginRight: theme.spacing(2),
-    [theme.breakpoints.only("xs")]: {
-      backgroundColor: "black",
-      borderRadius: theme.spacing(2),
-      marginRight: 0
-    }
-  }
 }));
 
 const Home: React.FC<Props> = ({}) => {
   const classes = useStyles();
-  const contentColor = "white";
 
   return (
-    <Grid container direction="row-reverse" className={classes.top}>
-      <Grid item className={classes.titleContent} sm={3}>
-        <Box pb={2} className={classes.contentWrapper}>
-          <Logo fill={contentColor} />
-          <Typography
-            className={classes.futuredays}
-            align="center"
-            variant="h5"
-          >
-            FutureDays
-          </Typography>
-        </Box>
+    <Box pt={16}>
+      <Grid container justify="center">
+        <Logo />
       </Grid>
-    </Grid>
+      <Box mt={12}>
+        <Grid container>
+          <Grid item container sm={3} justify="flex-start">
+            <NavButton type="about" text="OUR WORK" />
+          </Grid>
+          <Grid item container sm={6} justify="center">
+            <Container maxWidth="xs">
+              <Grid container justify="center" alignItems="center">
+                <Typography variant="body1">03.</Typography>
+                <Box mb={1}>
+                  <Typography variant="h4" className={classes.white}>
```

Comment:
Checkout the `gutterBottom` prop of Typography too

Reply:
Refrain from giving classnames a name which describes the css because you'll find you need to keep changing the class names when the styles change.

Call the class a name which describes the element, like :`classes.aboutUsHeader` or `mainheading`